.github/workflows/go.yml

```diff
@@ -30,3 +30,7 @@ jobs:
     - name: Test
       working-directory: claat
       run: go test -v ./...
+
+    - name: Make
+      working-directory: claat
+      run: make all release
```

Comment:
Or should this be using `-C` and/or `-j`, as it used to [here](https://github.com/googlecodelabs/tools/blob/9f2b585333fc582b0a545f6b4bee0f4466ef71c7/.travis.yml#L31)? (NB: #886)